jena-arq/src/test/java/org/apache/jena/riot/lang/TestLangJSONLD11.java

```diff
@@ -0,0 +1,55 @@
+package org.apache.jena.riot.lang;
+
+import static org.apache.jena.riot.Lang.JSONLD11;
+import static org.junit.Assert.assertEquals;
+
+import java.net.URI;
+
+import org.apache.jena.riot.RDFParserBuilder;
+import org.apache.jena.sparql.util.Context;
+import org.junit.Test;
+
+import com.apicatalog.jsonld.JsonLdError;
+import com.apicatalog.jsonld.JsonLdOptions;
+import com.apicatalog.jsonld.document.Document;
+import com.apicatalog.jsonld.loader.DocumentLoader;
+import com.apicatalog.jsonld.loader.DocumentLoaderOptions;
+import com.apicatalog.jsonld.loader.HttpLoader;
+
+public class TestLangJSONLD11 {
+
+    private static final String CONTENT = "";
+
+    @Test
+    public void testGetJsonLdOptions() {
+        RDFParserBuilder.create().context(setupContext()).lang(JSONLD11).fromString(CONTENT).build();
```

Comment:
This does not cause a parser to run.
RDFParserBuilder.build() returns the parser, not run it.

There are some terminating actions that do run the parser.

```
    @Test
    public void testGetJsonLdOptions() {
        StreamRDF sink = StreamRDFLib.sinkNull();
        RDFParser parser = RDFParserBuilder.create()
                .context(setupContext())
                .lang(JSONLD11)
                .fromString(CONTENT)
                .build();
        parser.parse(sink);

        assertEquals("Custom DocumentLoader wasn't called to handle loading", 1, TestDocumentLoader.COUNTER);
    }
```

Then CONTENT needs to be a valid document with an `"@context"` to trigger the document loader.